### ISSUE TITLE
Ctrl+w gets intercepted by firefox

### ISSUE THREAD

**Issue by WhyNotHugo** [glacambre/firenvim#1123]
- OS Version: linux x86-64
- Browser Version:  Firefox 91.0
- Browser Addon Version: 0.2.8
- Neovim Plugin Version: 0.2.8 (97f4794ffea6c559b71341597a293fe65863c3da)

### What I tried to do

When pressing ctrl+w, in insert mode, neovim deletes a word.

### What happened

When pressing ctrl+w, in insert mode on firenvim, firefox closes the tab.

I tried opening another issue about 7 times, but every time I try to delete a word memory muscle kicks in and I accidentally close the tab. :c

**Comment by glacambre** [comment]
Yes, as described [here](https://github.com/glacambre/firenvim#drawbacks) you need to patch firefox to be able to use `<C-w>`. This is a firefox issue that is unlikely to ever be fixed unfortunately.

**Comment by alerque** [comment]
In addition to the above linked Firefox patches that you use at compile time to build a patched version, there is also a way to hot-patch the binary after it's built: i.e. you can use binary distro packages as usual and then after each release whack out the nasty bit.

https://github.com/alerque/que-firefox/blob/master/bin/patch-firefox.sh

Note this has a hard coded path to the file on *my* system (and runs as root!) so you'd better have a look and adapt to your system rather than just blindly running my script.

**Comment by rwmpelstilzchen** [comment]
Thanks! This works like magic 🙂

If I’m not mistaken this is what you should use with LibreWolf:

```sh
#!/usr/bin/env sh
sudo perl -i -pne 's/reserved="false"/               /g' /usr/lib/librewolf/browser/omni.ja
find ~/.cache/librewolf/ -type d -name startupCache | xargs rm -rf
```

**Comment by SuggonM** [comment]
In Firefox forks that *do* allow remapping shortcuts (such as Zen), an easier way around would be to change the shortcut for Close Tab from <kbd>Ctrl</kbd> <kbd>w</kbd> to <kbd>Ctrl</kbd> <kbd>Shift</kbd> <kbd>w</kbd>. Will also save you from accidentally closing tabs out of habit :)

**Comment by glacambre** [comment]
Firefox lets you control what happens with ctrl+w through `about:keyboard`, so this issue can be closed.
